pymc3/distributions/multivariate.py

```diff
@@ -475,27 +475,6 @@ def __init__(self, a, transform=transforms.stick_breaking, *args, **kwargs):
 
         self.mode = tt.switch(tt.all(a > 1), (a - 1) / tt.sum(a - 1), np.nan)
 
-    def _random(self, a, size=None):
-        gen = stats.dirichlet.rvs
-        shape = tuple(np.atleast_1d(self.shape))
-        if size[-len(shape) :] == shape:
-            real_size = size[: -len(shape)]
-        else:
-            real_size = size
-        if self.size_prefix:
```

Comment:
`self.size_prefix` was only used in `random` method. So, I think we can remove its initialization.